app/src/main/java/com/example/myapplication/ui/ViewStates.kt

```diff
@@ -12,6 +12,7 @@ sealed interface ItemsHeight {
 data class ItemViewState(
     val title: String,
     val background: Color,
+    val widthRatio: Float = 1F,
```

Comment:
If it is needed only for 1 Composable then you need to create a new ViewState.

Reply:
Done